docs/about/index.html

```diff
@@ -74,6 +74,45 @@ <h1 class="article-title" itemprop="name">About</h1>
 <h3 id="github">Github</h3>
 
 <p><a href="https://github.com/packit-service/packit">https://github.com/packit-service/packit</a></p>
+
+<h3 id="the-key-principles-of-packit">The key principles of packit</h3>
```

Comment:
Is it intended to have this section in packit home page (<hostname>/) and also in about page (<hostname>/about)?

Reply:
Yes: we should have a decent landing page (/): newcomers should learn about packit as soon as possble: I hope that we achieve that with the principles.

I also like that we have the blog posts at /, so people know what the latest release is.

I'm also planning to add the "packit why" to the about page, probably only to /about.

WDYT?

Reply:
That sounds good :)